app/pages/lab/subject-sets.spec.js

```diff
@@ -29,7 +29,7 @@ describe('SubjectSetsPage', function () {
 
   it('will display a message when no subject sets are present', function () {
     const message = 'No subject sets are currently associated with this project.';
-    assert.equal(wrapper.find('p').text(), message);
+    assert.equal(wrapper.find('#no-subject-set').text(), message);
```

Comment:
I'd rather not have ids in the markup just for the tests. How about `wrapper.find('p').next()`?

Enzyme is using cheerio.js for DOM traversal, which is basically a lightweight jquery core lib: https://github.com/cheeriojs/cheerio